integration-tests/src/containers.rs

```diff
@@ -650,4 +662,86 @@ impl Redis {
             external_address,
         }
     }
+
+    pub fn pool(&self) -> deadpool_redis::Pool {
+        let redis_url = url::Url::parse(self.internal_address.as_str()).unwrap();
+        let redis_cfg = deadpool_redis::Config::from_url(redis_url);
+        redis_cfg
+            .create_pool(Some(deadpool_redis::Runtime::Tokio1))
+            .unwrap()
+    }
+
+    pub fn triple_storage(&self, id: &AccountId) -> mpc_node::storage::TripleStorage {
+        mpc_node::storage::triple_storage::init(&self.pool(), id)
+    }
+
+    pub fn presignature_storage(&self, id: &AccountId) -> mpc_node::storage::PresignatureStorage {
+        mpc_node::storage::presignature_storage::init(&self.pool(), id)
+    }
+
+    pub async fn stockpile_triples(&self, cfg: &NodeConfig, participants: &Participants, mul: u32) {
```

Comment:
I was expecting a storage solution. We can generate N trioles and pre-signatures and reuse them constantly. That will require hardcoding participant set, keys, etc. Not sure what is easier.

Reply:
Not sure what you mean by storage solution here? It's storing N*mul triples with this function per node.

Hardcoding is a whole slew of work there. The reason it's done this way is because presignatures are dependent on the initial key, which would mean we would then need to inject the key into the contract and each node would also need to intake their own share of the key.